typescript/packages/reality-capture-analysis/package.json

```diff
@@ -28,19 +28,15 @@
 	},
 	"license": "MIT",
 	"dependencies": {
-		"axios": "^1.6.0"
-	},
-	"peerDependencies": {
-		"@itwin/core-bentley": "^4.0.0",
-		"@itwin/core-common": "^4.0.0"
+		"axios": "^1.6.0",
+		"@itwin/reality-capture-common": "workspace:*",
+		"@itwin/reality-data-client": "workspace:*",
+		"@itwin/reality-data-transfer": "workspace:*"
 	},
 	"devDependencies": {
 		"@itwin/build-tools": "^4.0.0",
 		"@itwin/core-bentley": "^4.0.0",
```

Comment:
are you planning on removing the use of BentleyError & BentleyStatus?

Reply:
oops, will be fixed